TMS.ConsoleApp/Entities/Lecture.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.ConsoleApp.Entities
+{
+    internal class Lecture : EntityBase
+    {
+        public string Topic { get; set; } = string.Empty;
+        public PracticalLesson? PracticalLesson { get; set; }
+        public Lecture(string description, string topic, PracticalLesson practicalLesson)
```

Comment:
Lecture should not include Practice

> A lecture has a topic (an arbitrary string, possibly empty or equal to null),
>  and 
> a practical lesson - a link to a task condition (an arbitrary string, possibly empty or equal to null)

meaning that lecture has a topic and practice has a task